### PR TITLE
NIFI-13712 - Improve ListenSlack to catch channel join events

### PR DESCRIPTION
**Pull request by pvillard31**
# Summary

[NIFI-13712](https://issues.apache.org/jira/browse/NIFI-13712) - Improve ListenSlack to catch channel join events

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [ ] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [ ] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [ ] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [ ] Pull Request based on current revision of the `main` branch
- [ ] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 21

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-extension-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/ListenSlack.java [hunk 1]
```diff
@@ -96,6 +98,8 @@ public class ListenSlack extends AbstractProcessor {
         "The Processor is to receive only slack messages that mention the bot user (App Mention Events)");
     static final AllowableValue RECEIVE_COMMANDS = new AllowableValue("Receive Commands", "Receive Commands",
         "The Processor is to receive Commands from Slack that are specific to your application. The Processor will not receive Message Events.");
+    static final AllowableValue MEMBER_JOINED_CHANNEL_EVENTS = new AllowableValue("Member Joined Channel", "Member Joined Channel",
```
Comment (exceptionfactory) [inline nifi-extension-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/ListenSlack.java:101]: The pattern of the other values seems to be `Receive...` so from that perspective, should this be called `Receive Joined Channel Events`?

**Review by exceptionfactory — COMMENTED** [review]
Thanks for the improvement @pvillard31. The change looks straightforward, just one question on the value naming.

File: nifi-extension-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/ListenSlack.java (hunk 1 above)
Comment (pvillard31) [reply nifi-extension-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/ListenSlack.java]: Done, thanks @exceptionfactory !

**Review by exceptionfactory — APPROVED** [review]
Thanks for making the adjustments @pvillard31, looks good, +1 merging!

**Pull request merged**